doc/classes/SceneTree.xml

```diff
@@ -204,7 +204,7 @@
 	</methods>
 	<members>
 		<member name="current_scene" type="Node" setter="set_current_scene" getter="get_current_scene">
-			The current scene.
+			The topmost node of the current scene.
```

Comment:
We prefer to use "root" which is used everywhere in the documentation. "topmost" is not something used much currently.
```suggestion
			The root of the current scene.
```

Reply:
My only hesitation is that this change is to help clear up the confusion between `root` (which refers to the viewport root), `current_scene` (the current scene's root), and `editor_scene_root` (the editor scene's root). If you think this language is better suited for the description, I'm for it.

Reply:
I see, I missed the fact that there was possible confusion with the `root` property. That's a good point which would be worth disambiguating.

Reply:
Would you still recommend changing it to "root" or keep the "topmost node" proposed change? I'm in favor of the "scene currently edited in the editor" change either way.